README.md

```diff
@@ -1,26 +1,19 @@
-== README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
+[ ![Codeship Status for THP-nov-2016/up-all-knight](https://app.codeship.com/projects/4a444d90-8d9f-0134-2fd1-0298e52abb1b/status?branch=master)](https://app.codeship.com/projects/185059)
 
-* How to run the test suite
+# Up All Knight
+This application allows you to play a 1 vs 1 chess game online.
 
-* Services (job queues, cache servers, search engines, etc.)
+## Built with
+* [Ruby on Rails](http://rubyonrails.org/)
 
-* Deployment instructions
+## Authors
+* Joel Choi
```

Comment:
Check with the team before adding there's, but IMO email belongs here too:
```
* A Name <a_name@derp.com>
```

Or maybe link to Github profile instead.